rules/windows/builtin/ldap/win_ldap_recon.yml

```diff
@@ -1,14 +1,15 @@
-title: LDAP Reconnaissance / Active Directory Enumeration
+title: Potential Active Directory Reconnaissance/Enumeration
```

Comment:
I would like to keep "LDAP" in the title.

Reply:
Changed. Let me know what you think.